classes/WP_CLI/Action/Cancel.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use function \WP_CLI\Utils\get_flag_value;
+
+/**
+ * WP-CLI command: action-scheduler action cancel
+ */
+class ActionScheduler_WPCLI_Action_Cancel_Command extends ActionScheduler_WPCLI_Command {
+
+	/**
+	 * Execute command.
+	 *
+	 * @uses as_unschedule_action()
+	 * @uses as_unschedule_all_actions()
+	 * @uses $this->print_error()
+	 * @uses $this->print_success()
+	 * @return void
+	 */
+	public function execute() {
+		$hook          = '';
+		$group         = get_flag_value( $this->assoc_args, 'group', '' );
+		$callback_args = get_flag_value( $this->assoc_args, 'args', null );
+		$all           = get_flag_value( $this->assoc_args, 'all', false );
+
+		if ( ! empty( $this->args[0] ) ) {
+			$hook = $this->args[0];
+		}
+
+		if ( ! empty( $callback_args ) ) {
+			$callback_args = json_decode( $callback_args, true );
+		}
+
+		if ( $all ) {
+			$this->cancel_all( $hook, $callback_args, $group );
+			return;
+		}
+
+		$this->cancel_single( $hook, $callback_args, $group );
+	}
+
+	/**
+	 * Cancel single action.
+	 *
+	 * @param string $hook
+	 * @param array $callback_args
+	 * @param string $group
+	 * @return void
+	 */
+	protected function cancel_single( $hook, $callback_args, $group ) {
+		if ( empty( $hook ) ) {
+			\WP_CLI::error( __( 'Please specify hook of action to cancel.', 'action-scheduler' ) );
+		}
+
+		try {
+			$result = call_user_func( 'as_unschedule_action', $hook, $callback_args, $group );
+		} catch ( \Exception $e ) {
```

Comment:
Though they don't harm, we're qualifying our use of various types like this one unnecessarily. Here and elsewhere, I think if we can avoid unnecessary symbols, it becomes a little easier to read.

```suggestion
		} catch ( Exception $e ) {
```

Reply:
Going to leave this for now, for consistency (other AS classes use `\Exception`).